apps/wallet-connect/src/hooks/useWalletConnect.tsx

```diff
@@ -117,6 +117,19 @@ const useWalletConnect = () => {
               result = '0x'
               break
             }
+
+            case 'eth_signTypedData_v4': {
+              const [message, address] = payload.params
+
+              if (!areStringsEqual(address, safe.safeAddress)) {
+                throw new Error('The address or message hash is invalid')
```

Comment:
I know that this is a copypaste, so it's our bad. Could you remove `message hash` from the error message because only the address is checked?

Reply:
Done! Thanks for notifying~